src/server.c

```diff
@@ -3674,6 +3668,7 @@ void call(client *c, int flags) {
 
     server.fixed_time_expire--;
     server.stat_numcommands++;
+    prev_err_count = server.stat_total_error_replies;
```

Comment:
I have a question, why reset prev_err_count here.

Reply:
this is why this variable exists:
```c
    /* Update failed command calls if required.
     * We leverage a static variable (prev_err_count) to retain
     * the counter across nested function calls and avoid logging
     * the same error twice. */
```

maybe it would have been nicer to move that reset line to right next to the place it is used (incrementing `failed_calls`)